2-mandatory/1-weather-report.js

```diff
@@ -13,6 +13,16 @@
 
 function getTemperatureReport(cities) {
```

Comment:
Looks great!
A small point - keep an eye on indentation, as it will make it easier for other developers to read your code.